Source/AwsCommonRuntimeKit/Utilities.swift

```diff
@@ -57,3 +57,25 @@ extension Bool {
         return self ? 1 : 0
     }
 }
+
+func getMutablePointer<T>(memoryType: T?) -> UnsafeMutablePointer<T>? {
```

Comment:
Is it possible to write this as:
```swift
func fromPointer<T, P>(ptr: T?) -> P<T>? {
   if let ptr = ptr {
        let pointer = UnsafeMutablePointer<T>.allocate(capacity: 1)
        pointer.initialize(to: ptr)
        return P<T>(pointer)
    }
    return nil
}
```

Reply:
tried something similar to this but let me try again